code/modules/nifsoft/nifsoft.dm

```diff
@@ -242,7 +242,7 @@
 	stored = /datum/nifsoft/package/security
 
 /datum/nifsoft/package/security
-	software = list(/datum/nifsoft/ar_sec,/datum/nifsoft/flashprot)
+	software = list(/datum/nifsoft/ar_sec,/datum/nifsoft/cameramonitor,/datum/nifsoft/flashprot)
```

Comment:
I would not want the ability to access cameras available on a nif.

Reply:
As seen from testing, it only shows camera alarms

Reply:
Those are basically "Camera Alarm Monitor" so it will only show list of cameras that are broken and cameras with motion sensors detecting motion.  It will not allow to actually view cameras.

Reply:
It just allows you to check camera alarms, which is a feature already available on the AR-S glasses.

Reply:
Then I misunderstand what this is and also I shouldn't post on Github when I am half asleep. :U

But still I don't like the idea of a nif allowing you to be alerted to camera alarms.